shop/prestashop-performance/Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM prestashop/prestashop:1.7
+FROM prestashop/base:7.3-apache
+LABEL maintainer="devops team <devops@prestashop.com>"
+
+ENV PS_VERSION 1.7.7.0
+
+# Get PrestaShop
+ADD https://www.prestashop.com/download/old/prestashop_1.7.6.3.zip /tmp/prestashop.zip
```

Comment:
1.7.6.3 is not compatible with PHP 7.3.

https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart

Reply:
Good catch ! Will fix it.

Reply:
Thanks @PierreRambaud ;)